processor/geoipprocessor/README.md

```diff
@@ -39,6 +39,8 @@ The following settings must be configured:
 
 - `providers`: A map containing geographical location information providers. These providers are used to search for the geographical location attributes associated with an IP. Supported providers:
   - [maxmind](./internal/provider/maxmindprovider/README.md)
+- `source`:
```

Comment:
The configuration option `source.from` affects not only the source of the IP attribute, but also the destination where the geolocation attributes are created. Perhaps a better name for this option would be `context` with the values `resource` or `record`, with `record` meaning a log record, a metric data point, or a span, depending on signal? The [Transform processor](https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.107.0/processor/transformprocessor/README.md) uses the word `context` for similar concept: https://github.com/open-telemetry/opentelemetry-collector-contrib/blob/v0.107.0/processor/transformprocessor/README.md#contexts.

Reply:
Renaming done in https://github.com/open-telemetry/opentelemetry-collector-contrib/pull/34214/commits/fde278b85fd75fa152bc2a6d8ffa50d414867f0f

Context configuration looks more align to other Otel components :+1: